docs/ios/user-interface/controls/creating-tabbed-applications.md

```diff
@@ -301,12 +301,12 @@ There are a few important steps to note when adding a Storyboard to a previously
 	[![](creating-tabbed-applications-images/project-options.png "Set the Main Interface to MainStoryboard")](creating-tabbed-applications-images/project-options.png#lightbox)
 1. In your `App Delegate`, override the Window method, with the following code:
 
-	```csharp
-	public override UIWindow Window {
-	  get;
-	  set;
-	}
-	```
+```csharp
```

Comment:
Unfortunately there is no 'preview' of your changes :( but this code block still needs to be indented, so that it is rendered as part of the bullet above (ie. it keeps the bullet's indenting, not at all related to "code indenting"). Spaces are fine, however ;)